contributingGuides/CONTRIBUTING.md

```diff
@@ -58,6 +58,7 @@ Please be aware that compensation for any support in solving an issue is provide
 - Merged PR within 6 business days - 0% bonus 
 - Merged PR within 9 business days - 50% **penalty**  
 - No PR within 12 business days - **Contract terminated**  
+If the PR causes a regression within 7 days of being deployed to production, the bonus timeline restarts from the initial hire date. 
```

Comment:
This is how the text will look:

![Screenshot 2023-01-11 at 12 13 01 PM](https://user-images.githubusercontent.com/12268372/211752787-6c459ea7-c18c-4086-80d2-633976e658dd.png)

If we add a newline after the bullet list:
```suggestion
- No PR within 12 business days - **Contract terminated**  

If the PR causes a regression within 7 days of being deployed to production, the bonus timeline restarts from the initial hire date. 
```

It will look like:

![Screenshot 2023-01-11 at 12 14 31 PM](https://user-images.githubusercontent.com/12268372/211753035-3f57ea3f-6175-41cd-9eff-bdec9229fa5f.png)

Not a blocker, tho :)